Unix/etc/omiserver.conf

```diff
@@ -51,3 +51,4 @@ loglevel = DEBUG
 #configfile=PATH
 NoSSLv2=true
 #NoSSLv3=false
+useNtlmCredsFile=true

```

Comment:
Hmm, there is already a setting here. Seems like you can just decide if you should use it if it's defined or not ...